backend/api/views/user.py

```diff
@@ -0,0 +1,108 @@
+from flask import Flask, jsonify, request, Blueprint
+from api.models import User, db
+from api.core import create_response, serialize_list, logger
+import bcrypt
+
+user = Blueprint("user", __name__)
+
+# function that edits user model
+@user.route("/edit_user", methods=["POST"])
+def edit_user():
+    user_data = request.get_json()
+
+    try:
+        user_id = User.decode_auth_token(request.cookies.get("jwt"))
+    except:
+        return create_response(
+            message="Invalid token", status=401, data={"status": "fail"}
+        )
+
+    user = User.query.filter_by(id=user_id).first()
```

Comment:
I probably should've mentioned this before but the preferred way to get an object by ID in sqlalchemy is changing the above code to

`user = User.query.get(user_id)`
so you don't have to do the .first() or run a filter